CMakeLists.txt

```diff
@@ -177,7 +177,16 @@ endif()
 #------------------------------------------------------------------------------
 # install
 #------------------------------------------------------------------------------
-configure_file(mayaUSD.mod.template ${PROJECT_BINARY_DIR}/mayaUSD.mod)
+if (DEFINED MAYAUSD_TO_USD_RELATIVE_PATH)
+    set(USD_INSTALL_LOCATION "${CMAKE_INSTALL_PREFIX}/${MAYAUSD_TO_USD_RELATIVE_PATH}")
+else()
+    set(USD_INSTALL_LOCATION ${PXR_USD_LOCATION})
+endif()
```

Comment:
When given a USD relative path (from the ADSK internal build system) we will use that for the USD path in the .mod file.

Reply:
Note: "USD_INSTALL_LOCATION" is a variable inside the module template files.